xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editclass.vm

```diff
@@ -60,20 +60,24 @@ $xwiki.jsfx.use('js/xwiki/editors/dataeditors.js', true)##
 #macro (displayProperty $field)
   <div id="xproperty_${field.name}" class="xproperty #if($field.disabled)disabled#end">
     <div id="xproperty_${field.name}_title" class="xproperty-title ${field.type}">
-      <span class="toggle-collapsable" role="presentation">$services.icon.renderHTML('caret-down')</span>
-      <h2>
-        $!escapetool.xml($doc.displayView($field.xWikiClass.get('prettyName'), "${field.name}_" , $field))
-        ($doc.displayView($field.xWikiClass.get('name'), "${field.name}_" , $field):
+      <h3>
```

Comment:
Was a mistake before to use h2?

Reply:
I added additional structure to the page with a H2 : 
https://github.com/xwiki/xwiki-platform/pull/3254/files#diff-6188397dc36b434aca69e83161e131eb37f552008a22b918a6b8bd1bc925b22eR183-R189

This is clearly a subcategory of the propertylist, so I think a H3 here would be appropriate.